src/precice/SolverInterface.hpp

```diff
@@ -124,6 +135,13 @@ class SolverInterface {
   /**
    * @brief Advances preCICE after the solver has computed one timestep.
    *
+   * - Sends and resets coupling data written by solver to coupling partners.
+   * - Receives coupling data read by solver.
+   * - Computes and applied data mappings.
+   * - Computes acceleration of coupling data.
+   * - Exchanges and computes information regarding the state of the coupled
+   *   simulation.
```

Comment:
The order is a bit mixed.
It should be
* write mapping
* send
* coupling info and timestep
* accelerate (not sure about this one)
* receive 
* read mapping

Reply:
I don't read this as an ordered list. The order is also different depending on the coupling scheme and whether the first or second participant calls advance. From the outside perspective (`SolverInterface` and `SolverInterfaceImpl`) the order of the actions is an implementation detail. Therefore, I would leave it as it is.